packages/fxa-auth-db-mysql/lib/db/schema/patch-115-116.sql

```diff
@@ -0,0 +1,12 @@
+--
```

Comment:
uh oh, PR #8553 is also adding a migration so there will be a conflict.

Reply:
Thanks for pointing this out. If it's not merged by the time I'm ready to merge, I'll check with Vijay about it.